models/public/quartznet/accuracy-check.yml

```diff
@@ -0,0 +1,18 @@
+models:
+  - name: quartznet
+    launchers:
+      - framework: dlsdk
+        allow_reshape_input: True
+        adapter:
+          type: ctc_greedy_decoder
+          alphabet: [' ', a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, u, v, w, x, y, z, '''']
+        inputs:
+          - name: 'audio_signal'
+            type: INPUT
+            layout: NHWC
```

Comment:
are you sure in layout?
I believe input is 3-dim only, not 4

Reply:
The input is 3-dim but [accuracy_checker/launcher/dlsdk_launcher.py#L846](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py#L846) returns 4-dim `data_shape`. Do you have any suggestions to avoid such behavior?

Reply:
ok, then I will take it in account, I do not think that we should resolve this issue as part of this request